nullaway/src/test/java/com/uber/nullaway/tools/ErrorDisplay.java

```diff
@@ -50,16 +77,19 @@ public boolean equals(Object o) {
     }
     ErrorDisplay that = (ErrorDisplay) o;
     return type.equals(that.type)
-        // To increase readability, a shorter version of the actual message might be present in the
-        // expected output of tests.
+        && encMethod.equals(that.encMethod)
+        && encClass.equals(that.encClass)
+        && kind.equals(that.kind)
+        && clazz.equals(that.clazz)
```

Comment:
This is missing comparing `this.method`, which I believe is why that test case with the missing parenthesis succeeds...

Reply:
Thank you very much for noticing this, I just used the intellij generated code. updated it: 258a34176011f1f4373d9988f90f0ffd3742c7aa